src/Common/Query/GeocodeQuery.php

```diff
@@ -19,14 +19,14 @@
 /**
  * @author Tobias Nyholm <tobias.nyholm@gmail.com>
  */
-final class GeocodeQuery implements Query
+class GeocodeQuery implements Query
 {
     /**
      * The address or text that should be geocoded.
      *
      * @var string
      */
-    private $text;
```

Comment:
Is this needed? If you overwrite `withText` then there is no need for changing this to protected, right?

Reply:
Yes, I can overwrite all the methods to avoid changing the properties in the base class to from private to protected. But I thought the proper way was to change the visibilty of the properties, not to duplicate the code in the methods?